index.html

```diff
@@ -41,8 +41,167 @@ <h1 class="primary">
                </ul>
             </div>
             
-            <img src="assets/bottom arrow.svg" alt="bottom-arrow">
         </section>
+        
+        <div class="bottom-arrow">
+            <img src="assets/bottom arrow.svg" alt="bottom-arrow" >
+        </div>
+        
+        <section class="heading">
+            <h1 class="color1 primary">My Recent <br>Works</h1>
```

Comment:
Kindly I suggest you use the `h2` tag instead of the `h1` tag for the sub-heading for best practices. 😄